### PR TITLE
Develop fix tests #2

### PR DESCRIPTION
**Pull request by ilcsik**
@robertavram Added some tests for interventions as well. I think the tests are sufficient for now, 154 tests are passing now. We are only skipping the following ones:

- HACT related (2 tests),  which according to you is WIP
- V1 related (4 tests), which you suggested to skip for now
- StaffMember related (8 tests), which according to you is WIP
- t2f related, which Simon will able to fix after merge (6 tests)
- a few here or there like export or ref number (4 tests altogether)

### CODE REVIEW

File: EquiTrack/partners/views/interventions_v2.py [hunk 1]
```diff
@@ -102,29 +102,28 @@ def get_queryset(self, format=None):
...
             queries = []
 
-            # TODO: update these filters
-            if "partnership_type" in query_params.keys():
-                queries.append(Q(partnership_type=query_params.get("partnership_type")))
+            if "document_type" in query_params.keys():
+                queries.append(Q(partnership_type=query_params.get("document_type")))
             if "country_programme" in query_params.keys():
-                queries.append(Q(country_programme=query_params.get("country_programme")))
+                queries.append(Q(hrp__country_programme=query_params.get("country_programme")))
```
Comment (robertavram) [inline EquiTrack/partners/views/interventions_v2.py]: change this to agreement_country_programme

File: EquiTrack/partners/views/interventions_v2.py [hunk 2]
```diff
@@ -102,29 +102,28 @@ def get_queryset(self, format=None):
...
-                queries.append(Q(office__in=[query_params.get("office")]))
+                queries.append(Q(offices__in=[query_params.get("office")]))
             if "location" in query_params.keys():
-                queries.append(Q(locations__location=query_params.get("location")))
+                queries.append(Q(intervention_sector_locations__in=[query_params.get("location")]))
             if "search" in query_params.keys():
                 queries.append(
                     Q(title__icontains=query_params.get("search")) |
-                    Q(partner__name__icontains=query_params.get("search"))
+                    Q(agreement__partner__name__icontains=query_params.get("search"))
```
Comment (robertavram) [inline EquiTrack/partners/views/interventions_v2.py]: add number to search as well

File: EquiTrack/partners/views/interventions_v2.py [hunk 3]
```diff
@@ -249,4 +248,4 @@ def update(self, request, *args, **kwargs):
             instance = self.get_object()
             intervention_serializer = self.get_serializer(instance)
 
-        return Response(intervention_serializer.data)
\ No newline at end of file
+        return Response(intervention_serializer.data)
```
Comment (robertavram) [inline EquiTrack/partners/views/interventions_v2.py]: always add a newline in the end of the file

File: EquiTrack/partners/views/interventions_v2.py (hunk 1 above)
Comment (ilcsik) [inline EquiTrack/partners/views/interventions_v2.py]: Done in https://github.com/unicef/etools/pull/338/commits/970ee8d9e4db0ebf383b4c347f30ca2a11a6f806

File: EquiTrack/partners/views/interventions_v2.py (hunk 2 above)
Comment (ilcsik) [inline EquiTrack/partners/views/interventions_v2.py]: Done in https://github.com/unicef/etools/pull/338/commits/970ee8d9e4db0ebf383b4c347f30ca2a11a6f806

File: EquiTrack/partners/views/interventions_v2.py (hunk 3 above)
Comment (ilcsik) [inline EquiTrack/partners/views/interventions_v2.py]: Im not sure about this - Im up to date and I have a newline there. Also my editor has a setting and puts it there if I save the file.

Comment (robertavram) [inline EquiTrack/partners/views/interventions_v2.py]: must have been my end

File: EquiTrack/partners/views/interventions_v2.py [hunk 4]
```diff
@@ -102,29 +102,29 @@ def get_queryset(self, format=None):
...
+            if "start" in query_params.keys():
+                queries.append(Q(start__gte=query_params.get("start")))
+            if "end" in query_params.keys():
+                queries.append(Q(end__lte=query_params.get("end")))
             if "office" in query_params.keys():
-                queries.append(Q(office__in=[query_params.get("office")]))
+                queries.append(Q(offices__in=[query_params.get("office")]))
             if "location" in query_params.keys():
-                queries.append(Q(locations__location=query_params.get("location")))
+                queries.append(Q(intervention_sector_locations__in=[query_params.get("location")]))
```
Comment (robertavram) [inline EquiTrack/partners/views/interventions_v2.py]: try to put a name icontains on the location in the locations in the intervention_sector_locations. add a test on this as well if there isn;t one.

Comment (robertavram) [inline EquiTrack/partners/views/interventions_v2.py]: @ilcsik

Comment (ilcsik) [inline EquiTrack/partners/views/interventions_v2.py]: Done in https://github.com/unicef/etools/pull/338/commits/2e3647a2537a9b7cbeb37f35e64846009b37e4cd

**Pull request merged**